app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -34,7 +34,9 @@ $large-input-size: 50px;
   }
 }
 
-.gem-c-search__input[type="search"] { // overly specific to prevent some overrides from outside
+// stylelint-disable selector-max-id
+.gem-c-search__input[type="search"],
+#global-header .gem-c-search__input[type="search"] { // overly specific to prevent some overrides from outside
```

Comment:
Awful we need this but until we get rid of `govuk_template` styles we don't have a real alternative.

Reply:
Yes unfortunately we can't remove the conflicting styles from the `govuk_template` repo itself so till we get rid of that dependency, this will have to do 😬